composer.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "laravel/framework",
+    "name": "juliusza/framework",

```

Comment:
Why is this here? It is not your framwork...

Reply:
not suggesting to merge this